THREAD_README.txt

```diff
@@ -0,0 +1,62 @@
+THREAD README

```

Comment:
This feels like it should be added to the repository wiki

Reply:
Probably but we can make that a separate PR